docs/installation/Setup-OpenFunction-Serving-with-Knative.md

```diff
@@ -0,0 +1,216 @@
+# Setup OpenFunction Serving with Knative
+
+You can refer to the installation steps on the [Knative docs](https://knative.dev/docs/install/any-kubernetes-cluster/) for setup OpenFunction Serving. Or follow these steps:
```

Comment:
=>
You can refer to the steps in [Knative docs](https://knative.dev/docs/install/any-kubernetes-cluster/) to setup Knative or follow the steps below: